apis/buf.lock

```diff
@@ -0,0 +1,7 @@
+# Generated by buf. DO NOT EDIT.
```

Comment:
what's the expected workflow for this type of file being updated? is it something a developer will ever need to do with the `buf` tooling and include in a PR?

Reply:
Exceedingly rarely. The command is `buf mod update`, which I've mentioned in https://github.com/crossplane/crossplane/issues/4924 for now.